pact-jvm-consumer-kotlin/README.adoc

```diff
@@ -0,0 +1,156 @@
+= kPact - Kotlin idiomatic DSL for Pact
+:p: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Goals
+* Easy readable
+* Easy writable
+* Compile time safety (e.g. force mandatory data to be set)
+* short and concise (no boilerplate)
+* IDE support (e.g. auto-completion)
+* Things belonging to each other are in the same block
+* Things on the same logical level are on the same indentation level (auto-indentation of IDE)
+* Usable also from https://github.com/DiUS/pact-jvm/tree/master/pact-jvm-consumer-junit5[pact-jvm-consumer-junit5]
+  (from `PactDslWithProvider` to `RequestResponsePact`)
+* Support both:
+  1. prose like infix syntax (e.g. `"consumer" and "provider" havePact`)
+  2. fluent interface syntax (e.g. `consumer("consumer").hasPactWith("provider")`)
+
+== Examples
+
+[source,java,indent=0,role="primary"]
+.Infix DSL
+----
+include::src/test/kotlin/au/com/dius/pact/consumer/kotlin/dsl/examples/InfixDslExample.kt[lines=11..47]
```

Comment:
Github doesn't render this properly, which results in an unexpected experience. I expect the README to render correctly and have the complete content available. Having a short readme and linking to the github.io page is also fine.

Reply:
done
I've split into a brief README.md and a detailed, parsed asciidoc-generated gh-page linked in the readme.